Added minimap to level 15

Sean0z opened the pull request:
Added a minimap in Level 15

Sean0z committed:
Added minimap to level 15

Dominic-Roser reviewed (changes requested):
add to levels 9-16
the things we talked about 
- toggle on off in pause
- change the icons 
- seethroughish?

Sean0z committed:
Added minimap and toggle and adjusted lap counter position in levels 9-16

Sean0z commented:
- Added minimap to levels 9-16
- Adjusted location of lap counter just in case
- Added toggle on/off in pause menu
- I tried making it seethrough but it doesnt look very good

Dominic-Roser reviewed (approved):
Everything seems to be working, only thing is we might want to make the minimap bigger, bc currently all the info on the minimap is visible on the screen
